src/NuGet.Clients/VsConsole/PowerShellHost/PowerShellHost.cs

```diff
@@ -359,6 +358,10 @@ private void UpdateWorkingDirectory()
                     _solutionManager,
                     _deleteOnRestartManager);
 
+                var packagesConfigInstalled = new Dictionary<NuGetFramework, HashSet<PackageIdentity>>();
```

Comment:
Another thing, why you really need a dictionary here to maintain framework wise packages? eventually you create a hashset of packageIdentity only.